src/test/java/org/jenkinsci/plugins/workflow/EnvWorkflowTest.java

```diff
@@ -42,7 +42,7 @@ public class EnvWorkflowTest {
     @Rule public JenkinsRule r = new JenkinsRule();
 
     /**
-     * Verifies if NODE_NAME environment variable is available on a slave node and on master.
+     * Verifies if NODE_NAME environment variable is available on an agent node and on the built-in node.
```

Comment:
I know the PR same inclusive comments, but it feels like we should update the test output as well?  i.e. `echo \"My name on built-in node...'

Reply:
Pull request extended to include text changes in the test messages.